docs/user/openstack/README.md

```diff
@@ -176,68 +216,132 @@ Create and associate a floating IP to the ingress port:
 openstack floating ip create --port <ingress port id> <external network>
 ```
 
-Add A record in your dns for *apps. in your DNS:
-
-```
-*.apps.example.shiftstack.com  IN  A  <ingress FIP>
-```
-OR add A record in `/etc/hosts`:
-
-```
-    <ingress FIP> console-openshift-console.apps.example.shiftstack.com
-```
-
 ### Without Floating IPs
 
 If you don't want to pre-create a floating IP address, you will still want to create the API DNS record or the installer will fail waiting for the API.
 Without the floating IP, you won't know the right IP address of the server ahead of time, so you will have to wait for it to come up and create the DNS records then:
 
-    $ watch openstack server list
+```sh
+watch openstack server list
+```
 
 Wait for the `<cluster name>-api` server comes up and you can make your changes then.
+
 **WARNING:** The installer will fail if it can't reach the bootstrap OpenShift API in 30 minutes.
+
 Even if the installer times out, the OpenShift cluster should still come up. Once the bootstrapping process is in place, it should all run to completion.
 So you should be able to deploy OpenShift without any floating IP addresses and DNS records and create everything yourself after the cluster is up.
 
+## Running The Installer
+
+Finally, you can run the installer with the following command:
+
+```sh
+./openshift-install create cluster --dir ostest
+```
+
+**NOTE:** At the time of writing, once your `ingress-port` comes up, you will need to attach the apps FIP to it. First, find the ingress port like this:
```

Comment:
This is redundant with the steps documented in `API Access`. Perhaps we should just reference to it?